tests/run-make/comment-section/rmake.rs

```diff
@@ -0,0 +1,44 @@
+// Both GCC and Clang write by default a `.comment` section with compiler information.
+// Rustc received a similar .comment section, so this tests checks that this section
+// properly appears.
+// See https://github.com/rust-lang/rust/commit/74b8d324eb77a8f337b35dc68ac91b0c2c06debc
+
+//@ only-linux
+
+use std::path::PathBuf;
+
+use run_make_support::llvm_readobj;
+use run_make_support::rustc;
+use run_make_support::{cwd, env_var, read_dir, run_in_tmpdir};
+
+fn main() {
+    let target = env_var("TARGET");
+
+    run_in_tmpdir(|| {
```

Comment:
We don't need to use this, the tests now run in a temporary directory by default. So it's fine to write to (and read from) `cwd` :)